content/W2015/W15-event-destress.md

```diff
@@ -0,0 +1,20 @@
+Title: Big CSters Discussion Circles: De-Stressing
+Date: 2015-03-05 18:00
+Category: Events
+Tags: social
+Slug: de-stress
+Author: Anna Lorimer
+Summary: School is stressful! Come find out what strategies upper&ndash;year students use to handle their workload.
+
+Big and Little CSters, we would like to invite you to our first CSters discussion circle all about de&ndash;stressing!
+
+There will be tacos (vegetarian, gluten-free and vegan options are available) and other snacks! 
+
```

Comment:
This description is a bit short... maybe you could include a bit more about what the discussion circle will involve.